native/src/components/ListItem.tsx

```diff
@@ -55,7 +57,7 @@ type ListItemProps = {
 const ListItem = ({ language, title, thumbnail, children, Icon, navigateTo }: ListItemProps): ReactElement => (
   <StyledPressable onPress={navigateTo}>
     <ListItemView language={language}>
-      {!!thumbnail && <Thumbnail source={thumbnail} />}
+      {!!thumbnail && (React.isValidElement<SvgProps>(thumbnail) ? thumbnail : <Thumbnail source={thumbnail} />)}
```

Comment:
Should this check really be here? Can't this be moved to `SimpleImage` component?

Reply:
It can. But I don't think that's quite right. The SimpleImage component also manages styling, resizing, ... If we pass a react element this cannot be influenced anymore, as the thumbnail is already rendered. I'd prefer keeping it as it is. But I'll move the check to a utility punction to make this clearer. You're alright with this?